src/components/auth/AuthProvider.tsx

```diff
@@ -1,56 +1,54 @@
-import { useEffect, useState, useRef } from "react";
-import LayoutSkeleton from "../skeleton/Layout";
+import { useEffect, useState, useRef } from 'react';
+import { Layout as LayoutSkeleton } from '../skeleton/Layout';
 import {
-  OidcProvider,
-  OidcConfiguration,
-  TokenRenewMode,
-} from "@axa-fr/react-oidc";
-import { publicRequest, HTTP_VERBS } from "../../lib/commons/axios-utils";
+	OidcProvider,
```

Comment:
It seems there are config conflicts with indent codes. Using a linter could be useful